sdk/core/azure-core/src/main/java/com/azure/core/http/rest/ResponseConstructorsCache.java

```diff
@@ -132,6 +132,30 @@ private MethodHandle locateResponseConstructor(Class<?> responseClass) {
         }
     }
 
+    //TODO: add javadoc g2vinay
+    Response<?> invokeSync(final MethodHandle handle,
```

Comment:
Let's just make the `invoke` API synchronous, it never needed to be asynchronous